assets/dev/scss/frontend/_swiper-customisations.scss

```diff
@@ -7,8 +7,7 @@
         --swiper-pagination-bullet-size: 6px;
         --swiper-pagination-bullet-horizontal-gap: 6px;
 
-        .swiper,
-        .swiper-container {
+        :is( .swiper, .swiper-container ) {
```

Comment:
I have used the `:is()` selector to reduce the amount of css code that we generate.